Makefile

```diff
@@ -499,3 +499,6 @@ run-integration-tests:
 run-anvil: SHELL := /bin/sh
 run-anvil:
 	docker-compose -f integration-tests/docker-compose.anvil.yml up --remove-orphans
+
+run-integration-anvil:
```

Comment:
CI run won't finish because anvil will continue to run forever, we need to do something similar to https://github.com/status-im/status-go/blob/develop/Makefile#L492 with running silently and then reading exit code from the container with tests

Reply:
Updated.